bazarr.py

```diff
@@ -116,21 +116,12 @@ def should_stop(self):
 def start_bazarr(process_registry=ProcessRegistry()):
     script = [sys.executable, "-u", os.path.normcase(os.path.join(dir_name, 'bazarr', 'main.py'))] + sys.argv[1:]
 
-    ep = subprocess.Popen(script, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, stdin=subprocess.PIPE)
-    process_registry.register(ep)
     print("Bazarr starting...")
+    ep = subprocess.Popen(script, stdout=None, stderr=None, stdin=subprocess.DEVNULL)
```

Comment:
To be fair, from what I see in the documentation `None` is the default value for `stdout` and `stderr` so I could just remove it.